data/json/items/gun/223.json

```diff
@@ -1386,5 +1386,76 @@
       }
     ],
     "melee_damage": { "bash": 10 }
+  },
+  {
+    "id": "rdb_223",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "5.56x45mm bullpup rifle" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the AR-15 platform and a 5.56x45mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rdb_223",
+        "name": { "str_sp": "Kel-Tec RDB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines from the AR-15 platform and the 5.56 NATO cartridge.  The name stands for rifle, downward-ejection, bullpup, and its design was intended to incorporate some technical solutions, such as downward case ejection and convenience for left-handers, to solve some problems with bullpup guns.  However, the good idea's execution is not as perfect as the manufacturer (and user) would like.  No good idea from Kel-Tec is immune to Kel-Tec manufacturing issues, sadly.",
+        "weight": 1
+      }
+    ],
+    "weight": "3200 g",
+    "volume": "3421 ml",
+    "longest_side": "693 mm",
+    "barrel_length": "439 mm",
+    "price": 100000,
+    "price_postapoc": 5500,
+    "to_hit": -2,
+    "material": [ "steel", "aluminum", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "223" ],
+    "dispersion": 500,
```

Comment:
Sight_dispersion, not dispersion. Your original dispersion was fine, but adding a separate sight_dispersion entry will let it be overridden when other sights are installed.